nameko_amqp_retry/backoff.py

```diff
@@ -44,7 +48,7 @@ def get_next_schedule_item(cls, index):
             item = cls.schedule[index]
         return item
 
-    def get_next_expiration(self, message, backoff_exchange_name):
+    def calculate_next_expiration(self, message, backoff_exchange_name):
```

Comment:
I like `next()` and mirroring the iterator interface. You could actually bring back the return statement then.